tensorflow/compiler/xla/service/gpu/llvm_gpu_backend/gpu_backend_lib.cc

```diff
@@ -584,18 +640,29 @@ StatusOr<std::vector<uint8>> EmitModuleToHsaco(
   std::string tempdir_name = tempdir_vector.front();
```

Comment:
Would it be possible (and better?) to pick a new sub-directory each time, instead of different file names?

Reply:
Not sure what's the benefit in that. More library calls, and all the files are going to be deleted anyway. Also, there's a theoretical possibility that we won't be able to delete the subdirectory if it is still open in some subprocess when we get to the end of the function. 
In any event, that is an additional feature and shouldn't go into this PR, I think.